src/test/java/uk/co/tfn/ChromeTestCase.java

```diff
@@ -252,6 +226,34 @@ public void chromePeriodMultipleServices() throws IOException {
         assertTrue(isUuidStringValid(driver));
     }
 
+    @Test
+    public void chromePeriodMultipleProducts() throws IOException {
+        
+        getHomePage(driver);
+
+        waitForPageToLoad(driver);
+
+        stepsToPeriodPage(driver);
+
+        driver.findElement(By.id("periodtype-single-set-service")).click();
+
+        continueButtonClick(driver);
+
+        randomlyChooseAndSelectServices(driver);
+
+        continueButtonClick(driver);
+
+        Integer numberOfProducts = 4;
```

Comment:
I think this can be an int, thats the primitive in Java whereas Integer is a wrapper around int. I think we would only use Integer if the number was more complex.